assets/foliate-js/src/book.js

```diff
@@ -1167,7 +1167,22 @@ class Reader {
   }
 
   #onTouchEnd = ({ detail: e }) => {
-    if (this.#ignoreTouch()) return;
+    if (this.#ignoreTouch()) {
+      if (e.touchState.direction === 'vertical') {
+        const renderer = this.view.renderer;
+        const scrollTop = renderer.shadowRoot.querySelector('#container').scrollTop;
```

Comment:
On Android, when the user is at the top of the page, `scrollTop` is 0px.  
On iOS, due to rubber banding, it can be negative (eg -150px).  
`renderer.start` uses `Math.abs()` and would therefore return 150px instead of the expected -150px.  
Reading `scrollTop` directly makes it possible to correctly detect whether the user is at the top of the page.